core/server/api_container/server/startosis_engine/startosis_packages/package_replace_options_repository/package_replace_options_repository.go

```diff
@@ -0,0 +1,12 @@
+package package_replace_options_repository
+
+type PackageReplaceOptionsRepository struct {
+}
+
+func NewPackageReplaceOptionsRepository() *PackageReplaceOptionsRepository {
```

Comment:
Is anyone using this?

Reply:
oh! yes it cames from the previous approach, I'll remove it, good shout!